scico/flax/inverse.py

```diff
@@ -262,7 +262,8 @@ class ODPProxDcnvBlock(Module):
     def setup(self):
         """Computing operator norm and setting operator for batch
         evaluation and defining network layers."""
-        self.operator_norm = operator_norm(self.operator)
+        self.operator_norm = jnp.sqrt(power_iteration(self.operator.H @ self.operator)[0].real)
+
```

Comment:
If this is used more than once, it may be worth also duplicating `operator_norm` within `scico.flax`.

Reply:
Right now it is used only once. If we add other models that require this functionality, we will revisit the point of duplicating `operator_norm`.